### PR TITLE
Convert remaining endpoints to assumerole pattern, cleanup

### PR DESCRIPTION
**Pull request by fishnix**
* Move the remaining `/space` and `/metrics` routes to use the established assumeRole pattern
* Adapt the `/space` logic into our orchestrator pattern
* Remove deprecated routes 
  * metrics routes on the cost router
  * cost per resource route, in favor of dimensions
* Cleanup Readme

### CODE REVIEW

**Comment by btassone** [comment]
Following up per our conversation on slack. Don't forget to update the config.example.json and remove the "accounts" section in favor of just using "account" and adding the accounts map bit as well (maybe).

**Comment by btassone** [comment]
Everything else looks good! Routes all seem to work

**Review by btassone — APPROVED** [review]

File: docker/config.deco.json [hunk 1]
```diff
@@ -19,6 +19,10 @@
     "externalId": "{{ .external_id }}",
     "role": "{{ .cross_account_role }}"
   },
+  "accountsMap": {
```
Comment (tenyo) [inline docker/config.deco.json:22]: I don't think we need the list of `accounts` above anymore

**Review by tenyo — COMMENTED** [review]
Looks good 👍

File: api/handlers_spaces.go [hunk 2]
```diff
@@ -4,240 +4,54 @@ import (
...
+		handleError(w, apierror.New(apierror.ErrInternalError, "failed to generate policy", err))
 		return
 	}
-	log.Debugf("found cost explorer result cache %+v", *resultCache)
 
-	// vars for checking input times parse and are valid
-	var start string
-	var end string
-	var err error
+	orch, err := s.newCostExplorerOrchestrator(r.Context(), &sessionParams{
```
[the remark this answers is not in the document]
Comment (fishnix) [reply api/handlers_spaces.go:32]: yeah nice pattern :)

**Pull request merged**